aemedge/blocks/popular-blogs/popular-blogs.css

```diff
@@ -1,12 +1,183 @@
+    width: 100%;
+}
+
+.popular-blogs .heading-wrapper {
+    display: flex;
+    font-size: 42px;
+    justify-content: center;
+    flex-flow: wrap column;
+    align-items: center;
+}
+
+.popular-blogs .heading-wrapper img {
+    max-height: 36px;
+    max-width: 36px;
+    vertical-align: middle;
+}
+
+.popular-blogs .headline {
+    color: #1d1d1d;
+    font-size: 20px;
+    font-weight: bold;
+    line-height: 30px;
+    margin: 7px auto 44px;
+    max-height: 30px;
+    text-align: center;
 }
 
-.popular-blogs{
+.popular-blog-title {
+    color: #000a12;
+    font-size: 16px;
+    font-weight: bold;
+    line-height: 26px;
+    max-width: 288px;
+    max-height: 164px;
+}
+
+.popular-blogs .blog-slides {
     display: flex;
+    scroll-behavior: smooth;
+    scroll-snap-type: x mandatory;
+    overflow: scroll clip;
+    margin: 0;
+    padding: 0;
+}
+
+.popular-blogs .blog-slides::-webkit-scrollbar {
+    display: none;
+}
+
+.popular-blogs .blog-slide {
+    flex: 0 0 100%;
+    scroll-snap-align: start;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
     justify-content: center;
+    position: relative;
+    width: 100%;
+}
+
+.popular-blogs .blog-slide .blog-link {
+    display: flex;
+    flex-flow: wrap column;
+}
+
+.popular-blogs .blog-slide img {
+    max-height: 163px;
+    max-width: 288px;
+    margin-bottom: 12px;
+    border-radius: 4px;
+    box-shadow: 0 5px 15px 0 rgb(0 0 0 / 14%);
+    vertical-align: middle;
+    border: 0;
+}
+
+.popular-blogs .slider-nav-container {
+    position: absolute;
+    left: calc(100% / 2 - 180px);
+    right: calc(100% / 2 - 180px);
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    z-index: 100;
+}
+
+.popular-blogs .slider-nav-container button.slide-prev,
+.popular-blogs .slider-nav-container button.slide-next {
+    all: unset;
+    background: 0;
+    border: 0;
+    color: #171725;
+    height: 164px;
+    outline: 0;
+}
+
+.popular-blogs .slider-nav-container button.slide-prev>span,
+.popular-blogs .slider-nav-container button.slide-next>span {
+    padding: 1px 6px;
+}
+
+.popular-blogs .slider-nav-container span.fa {
+    display: inline-block;
+    font: normal normal normal 14px / 1 var(--font-awesome);
+    font-size: inherit;
+    text-rendering: auto;
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+}
+
+.popular-blogs .slider-nav-container span.fa-chevron-right::before {
+    content: "\f054";
+}
+
+.popular-blogs .slider-nav-container span.fa-chevron-left::before {
+    content: "\f053";
+}
+
+@media (width >=768px) {
+    .section.popular-blogs-container {
+        max-width: 1440px;
+    }
+
+    .popular-blogs .slides-container {
+        display: flex;
+        justify-content: center;
+        flex-flow: wrap column;
+        overflow: auto
+    }
+
+    .popular-blogs .blog-slides {
+        display: flex;
+        margin: 0 auto;
+        justify-content: center;
```

Comment:
I think you will need to remove "justify-content: center" too so that the scrollbar starts at the left-most card.